utils/crypto/envVars.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# equivalent of a __dirname in node.js
+DIR=$(dirname "${BASH_SOURCE[0]}")
+
+# makes sure which node_modules to use
+MOD_DIR="$DIR/../../node_modules"
+
+# workaround for an error with sourcing
+TS_NODE=$(yarn --modules-folder $MOD_DIR bin ts-node)
```

Comment:
OT: why is this needed?

Reply:
Since this file is sourced inside of another project, `yarn ts-node` might not be available in `node_modules`. So we make sure that we use the one that is in the root `node_modules`.